apps/react-app/src/common-app/components/export-config/export-config.component.tsx

```diff
@@ -24,11 +26,15 @@ export const ExportConfig: React.FC<Props> = props => {
     onHTMLSettingSelectionChanged(htmlTemplate, { primaryColor: event.target.value });
   };
   const handleExportConfigSelection = () => {
+    setIsDownload(true);
     onExportToHTML(exportHTMLSettings);
+
+    setTimeout(() => {
+      cancelExport();
+    }, 2500);
```

Comment:
Saco este valor a un constante fuera del componente en el mismo fichero

_DOWNLOAD_MESSAGE_TIMEOUT_ 2500

Reply:
is fixed